server/src/schemas/reservationsSchema.ts

```diff
@@ -0,0 +1,66 @@
+import { gql } from "apollo-server-express";
+
+/*
+Potential future enhancements:
+  - Attach files to reservations
+  - Remove or edit reservation comments
+  - Reschedule reservations
+ */
+
+export const ReservationsSchema = gql`
+  enum ReservationStatus {
+    PENDING
+    CONFIRMED
+    CANCELLED
+  }
+
+  type Reservation {
+    id: ID!
+    maker: User
+    labbie: User
+    createDate: DateTime
+    creator: Person
```

Comment:
Tangential but why do we have both `User` and `Person` objects if a `User` is generic enough to be used for anyone in the system? Also we'll have their `type` and `privilegeLevel` known as well whereas the `Person` object as it is implemented stores very little information about the person. Is it for unregistered students to be able to make reservations? I thought Mike said they need to register first even if it's a walk-in but I might be wrong

Reply:
My bad - I meant to do `creator: User` here. I agree that `Person` is redundant now. I pinged @pjw9963 and he agrees, so we can remove the `Person` type in a future PR.